### PR TITLE
Ability to increase font size/line spacing. #25

### PR DESCRIPTION
**Pull request by Antogin**
Added a way to change font and line spacing,
I changed the settings model and added the style with the ngStyle directive
This is my first ever contribution to an open source project, feedack appreciated !
:)

screenshot from 2017-04-18 14-56-11
screenshot from 2017-04-18 14-56-31
screenshot from 2017-04-18 14-56-55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,3 +101,4 @@ A million thanks to some awesome people :)
 * [Zach Berger](https://github.com/zachberger)
 * [blAck PR](https://github.com/blackpr)
 * [Bram Borggreve](https://github.com/beeman)
+* [Antonio Indrianjafy](https://github.com/Antogin)
```
Comment (housseindjirdeh) [inline README.md]: :tada:

File: src/app/core/settings/settings.component.html [hunk 2]
```diff
@@ -43,6 +43,28 @@
 						Black (AMOLED)
 					</label>
 				</div>
+          <div>
```
Comment (housseindjirdeh) [inline src/app/core/settings/settings.component.html]: Let's add an `h2` header here before these elements. Something like 'Change Font' should be okay

File: src/app/core/settings/settings.component.ts [hunk 3]
```diff
@@ -29,4 +29,14 @@ export class SettingsComponent implements OnInit {
   selectTheme(theme) {
     this._settingsService.setTheme(theme);
   }
+
+  changeTitleFont(val){
+    console.log(val);
```
Comment (housseindjirdeh) [inline src/app/core/settings/settings.component.ts]: Did you forget these console logs? :)

File: src/app/shared/models/settings.ts [hunk 4]
```diff
@@ -2,4 +2,6 @@ export interface Settings {
    showSettings: boolean;
    openLinkInNewTab: boolean;
    theme: string;
+   titleFontSize: string;
```
Comment (housseindjirdeh) [inline src/app/shared/models/settings.ts]: We can't save integers to localStorage can we? Strings are fine if that's the case :+1:

File: src/app/core/settings/settings.component.ts [hunk 5]
```diff
@@ -29,4 +29,14 @@ export class SettingsComponent implements OnInit {
...
+    this._settingsService.setFont(val);
+  }
+
+  changeSpacing(val){
+    console.log(val);
```
Comment (housseindjirdeh) [inline src/app/core/settings/settings.component.ts]: Here as well

**Comment by housseindjirdeh** [comment]
@Antogin aside from a few small things, this is awesome :)

Thank you, thank you and thank you. This is such a good first time contribution and I genuinely can't thank you enough for this :raised_hands:

File: src/app/core/settings/settings.component.ts (hunk 5 above)
Comment (Antogin) [inline src/app/core/settings/settings.component.ts]: On it !

File: src/app/shared/models/settings.ts (hunk 4 above)
Comment (Antogin) [inline src/app/shared/models/settings.ts]: Yea unfortunately

**Comment by housseindjirdeh** [comment]
LGTM :tada: :tada:

**Pull request merged**